src/SigningPortal/SigningPortal.tsx

```diff
@@ -103,22 +116,45 @@ export const SigningPortal: React.FC = () => {
     const loadChainProperties = async () => {
       if (client) {
         let chainSpec = await client.getChainSpecData();
-        if (chainSpec.properties.ss58Format && chainSpec.properties.tokenDecimals && chainSpec.properties.tokenSymbol) {
-          setChainProperties(chainSpec.properties)
-        }
+        // Extract properties & provide default values for missing ones
+        let updatedProperties: ChainProperties = {
```

Comment:
I had to fix the way chain properties are set because, in some cases, `tokenSymbol` is missing, preventing the properties from updating correctly. Now, the update ensures that missing values are handled properly.